File: Tareas/15/Código RUBY/tarea15.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

print("Ingrese el numero de terminos: ")
n = gets().to_i

serie = ""
i = 1
for i in(1...n)
	a = 3.1416*i

	serie = serie + a.to_s + ", "
end

a = 3.1416*i
serie = serie + a.to_s + "."

print(serie)

File: Tareas/15/Código JAVA/tarea15.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea15{
	public static void main (String[] args){
		int i, n;
		double a;
		String entrada;
		String serie = "";

		entrada = JOptionPane.showInputDialog("Ingrese el numero de terminos: ");
		n = Integer.parseInt(entrada);

		for(i=1; i<n; i++){
			a = 3.1416*i;
			serie = serie + a + ", ";
		}
		a = 3.1416*i;
		serie = serie + a + ".";
		JOptionPane.showMessageDialog(null,serie);
	} //Fin de función main
} //Fin de clase tarea15